docs/UserGuide.md

```diff
@@ -63,6 +63,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* If you are using a PDF version of this document, be careful when copy-pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
```

Comment:
Imo, is quite odd to put this under the **"notes about the command format"** section as it does not fit well with the other pointers in it. Perhaps, it will be better to be put it in a separate section like the FAQ?

Reply:
In my opinion, positioning this information here is beneficial as it pertains directly to the use of commands. From personal experience, I don't typically read through the entire User Guide before starting to use the commands. Therefore, having notes about potential pitfalls or issues within the command format section can be extremely useful.

Furthermore, I view the FAQ as an additional, somewhat less critical feature, which should not interfere with or impede the fundamental understanding and usage of the commands. As such, I believe it's acceptable to prioritize the 'Note about the command' section over the FAQ.

Reply:
Thanks for inputs @jundatan @SPWwj 
Yes it is a bit odd to have in under 'notes about the command format'. Just below it as a separate box is one viable option. Putting it in the FAQ is not a good idea, as @SPWwj mentioned. For now, let's keep it in the current place as it is less disruptive to the flow of the document.